IdentityCore/src/workplacejoin/MSIDPkeyAuthHelper.m

```diff
@@ -146,4 +150,16 @@ + (NSString *)createDeviceAuthResponse:(NSString *)audience
     return [MSIDJWTHelper createSignedJWTforHeader:header payload:payload signingKey:[identity privateKeyRef]];
 }
 
++(void)saveTelemetryForAdfsPkeyAuthChallengeForUrl:(NSURL *)submitUrl
```

Comment:
Per code above this is authorizationEndpoint and not specifically SubmitURL. Maybe rename the argument?

Reply:
Thanks, renamed it.